backend/docker-compose.yaml

```diff
@@ -45,65 +99,51 @@ services:
     image: amplicalabs/content-publishing-service:latest
     platform: linux/amd64
     ports:
-      - 3001:3000
-    env_file:
-      - path: .env.common.docker
-        required: true
-      - path: '.env.content-publishing-service'
-        required: false
+      - 3010:3000
```

Comment:
Do we need to expose the ports for the microservices at all? All the traffic should be in the docker network except for social-app-template-backend. 🤔

Reply:
If the SAT is running bare-metal, then the ports for the Gateway services inside Docker need to be exposed to the local host.

For a completely containerized setup, where SAT backend is also running inside Docker, then only the SAT backend port & the IPFS port would need to be exposed (because the frontend needs those)